workflow/models.py

```diff
@@ -94,9 +94,8 @@ class Issue(models.Model):
         (RESOLVED, _('Resolved'))
     )
     root = models.ForeignKey('self', null=True, blank=True)
-    project = models.ForeignKey(Project, verbose_name=_('Project'))
-    sprint = models.ForeignKey(Sprint, verbose_name=_('Sprint'),
-                               null=True, blank=True)
+    project = models.ForeignKey(Project, null=True, blank=True)
```

Comment:
Wrong project configuration. Use the models.ForeignKey(Project, verbose_name=_('Project'))